<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_room</class>
 <widget class="QWidget" name="add_room">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="numberofroom">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>87</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Area">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>187</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
color:rgb(238, 238, 236);</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>275</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>301</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>326</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="enter">
   <property name="geometry">
    <rect>
     <x>123</x>
     <y>355</y>
     <width>170</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="bathroom">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>64</y>
     <width>96</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="kitchen">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>90</y>
     <width>96</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="living">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>115</y>
     <width>96</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="warning_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>417</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pich.qrc">:/picha/Pichures/add_room.png</pixmap>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>numberofroom</zorder>
  <zorder>Area</zorder>
  <zorder>checkBox_2</zorder>
  <zorder>checkBox_3</zorder>
  <zorder>checkBox_4</zorder>
  <zorder>enter</zorder>
  <zorder>bathroom</zorder>
  <zorder>kitchen</zorder>
  <zorder>living</zorder>
  <zorder>warning_2</zorder>
 </widget>
 <resources>
  <include location="pich.qrc"/>
 </resources>
 <connections/>
</ui>
